lib/routes/sms.js

```diff
@@ -83,11 +86,11 @@ module.exports = (log, config, customs, sms) => {
         }
 
         function sendMessage (senderId) {
-          return sms.send(phoneNumber, senderId, messageId, acceptLanguage)
+          return sms.send(phoneNumber, senderId, templateName, acceptLanguage)
         }
 
         function logSuccess () {
-          return request.emitMetricsEvent(`sms.${messageId}.sent`)
+          return request.emitMetricsEvent(`sms.${templateName}.sent`)
```

Comment:
Good point. I started to do this as part of this PR just now, but it started to get out of hand as I had to move big chunks of the document round, add extra sections, and started documenting things like the view names too. Now I'm thinking I should maybe do all that together in a separate documentation PR?